CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to Snoopy
+
+This doc outlines the architecture of Snoopy and how provides a step-by-step
+guide of setting it up on your machine.
+
+## Setup
+
+Snoopy is a [Yarn workspace](https://yarnpkg.com/en/docs/workspaces) that is
+composed of several separate packages. These packages are all published to NPM
+together with [Lerna](https://github.com/lerna/lerna). First install all
+dependencies with:
+
+```
+yarn
+```
+
+To build Snoopy for development, you need to compile the codebase with
+[TypeScript](https://www.typescriptlang.org/). In the root of the project run:
+
+```
+yarn build:watch
+```
+
+This will watch and incrementally build all packages. If you only want to build
+without watching, drop the `:watch`.
+
+## Running on an app
+
+Snoopy itself is useless without an App to run it on. There are several examples
```

Comment:
can't you run Snoopy on Snoopy?

Reply:
You can by running it from the `packages/ui` directory. I'll add a note of this